samples/Makefile

```diff
@@ -340,7 +340,7 @@ endif
 
 define SUBDIR_recipe
 
-@$(MAKE) -C $(dir) --no-print-directory $@
+@+$(MAKE) -C $(dir) --no-print-directory $@
```

Comment:
Why the + ?

Reply:
It blocks the `-j2` in subdirectories.  Without that `+`, we get a warning message:  `Jobserver unavailable. Use '+' in parent makefile rule.`